src/test/java/org/broadinstitute/hellbender/utils/fasta/CachingIndexedFastaSequenceFileUnitTest.java

```diff
@@ -1,9 +1,7 @@
 package org.broadinstitute.hellbender.utils.fasta;
```

Comment:
I had accidentally changed/moved it but subsequently placed it back in its place.